client/my-sites/media/main.jsx

```diff
@@ -86,20 +89,27 @@ class Media extends Component {
 	};
 
 	closeDetailsModal = () => {
+		const site = this.props.selectedSite;
 		this.setState( {
 			editedImageItem: null,
 			editedVideoItem: null,
 			currentDetail: null,
 			selectedItems: [],
 		} );
+		if ( this.isSingleMediaView() ) {
```

Comment:
These 3 lines of code are still repeated ;) perhaps it would make sense to move them all to a separate method, vs having only `isSingleMediaView()` in a separate method.